docs/development.md

```diff
@@ -0,0 +1,133 @@
+﻿# FrankenPHP Development Environment Setup
+
+---
+
+## For Windows: WSL2 Setup
+
+1. Install WSL2:
+
+   ```powershell
+   wsl --install
+   ```
+
+2. Install a Linux distribution (example: AlmaLinux 10) via e.g. `wsl --install AlmaLinux-10`.
```

Comment:
What's AlmaLinux 10? Is it the official recommandation for WSL? If not, maybe we should discuss what's the "official" recommended distribution to do this